.circleci/config.yml

```diff
@@ -1,48 +1,6 @@
 version: 2
 
 jobs:
-  build_and_test_python27:
-    docker:
-      - image: circleci/python:2.7
-      - image: rabbitmq:3
-      - image: mongo:3.4
-
-    working_directory: ~/repo
-
-    environment:
-      VIRTUALENV_DIR: "~/virtualenv"
-      # Don't install various StackStorm dependencies which are already
-      # installed by CI again in the various check scripts
-      ST2_INSTALL_DEPS: "0"
-
-    steps:
-      - checkout
-      - restore_cache:
-          key: v1-dependency-cache-py27-{{ checksum "requirements.txt" }}
-      - run:
-          name: Download dependencies
-          command: |
-            git clone -b master git://github.com/stackstorm-exchange/ci.git ~/ci
-            ~/ci/.circle/dependencies
-      - run:
-          name: Run tests (Python 2.7)
-          command: ~/ci/.circle/test
-      - save_cache:
-          key: v1-dependency-cache-py27-{{ checksum "requirements.txt" }}
-          paths:
-            - ~/.cache/pip
-            - ~/.apt-cache
-      # NOTE: We use virtualenv files from Python 2.7 step in "deploy" job so we
-      # only persist paths from this job
-      - persist_to_workspace:
-          root: /
-          paths:
-            - home/circleci/ci
-            - home/circleci/virtualenv
-            - tmp/st2
-            - home/circleci/repo
-            - home/circleci/.gitconfig
-
   # NOTE: Until we add "python_version" metadata attribute to pack.yaml and
   # explicitly call which packs work with Python 3.x, Python 3.x failures are
   # not considered fatal
```

Comment:
Is there a need to adjust this note too?

Reply:
Removed, good catch.